src/App.js

```diff
@@ -26,17 +27,18 @@ function App() {
       const data = await response.json();
       const todos = data.records.map((todo) => {
         return { title: todo.fields.Title, id: todo.id, createdAt: todo.createdTime }
-      })
-      const newTodos = todos.sort((a, b) => new Date(a.createdAt) - new Date(b.createdAt))
-      setTodoList(newTodos)
+      }).sort((a, b) => new Date(a.createdAt) - new Date(b.createdAt))
+
+      setTodoList(todos)
       setIsLoading(false)
 
     } catch (error) {
       console.log(error.message)
     }
   }
 
-  const addTodo = async (url, data) => {
+  const addTodo = async (data) => {
+    const url = `https://api.airtable.com/v0/${process.env.REACT_APP_AIRTABLE_BASE_ID}/${process.env.REACT_APP_TABLE_NAME}`
```

Comment:
Nitpick:
Since this `url` variable is the same url for all of the CRUD operations, I'd recommend extracting it into a global variable. For example, define this url above the component and reference it within your callback functions:
```js
const API_URL = `https://api.airtable.com/v0/${process.env.REACT_APP_AIRTABLE_BASE_ID}/${process.env.REACT_APP_TABLE_NAME}`

...etc

const onAddTodo = async (data) => {
  try {
     const response = await fetch(API_URL, { ... });
   } catch (err) {
       ...etc
   }
}
```

The upside to this approach is if you change this `API_URL`, it affects all CRUD requests with 1 change (rather than having to update the individual `url` variable for each callback).